.npmrc

```diff
@@ -0,0 +1 @@
+tag-version-prefix=""
```

Comment:
Noticed the past tags didn't have the prefix, so I added config for this.

Reply:
thanks!